lib/main.js

```diff
@@ -394,7 +394,7 @@ define(['./errors'], function(errors) {
     // for easy membership testing with the "in" keyword
     function setLike(list, fill /* default true*/) {
         if(fill === undefined) fill = true;
-        var setLike = {};
+        var setLike = Object.create(null);
```

Comment:
mostly due to the fact that in JS objects (and arrays) are considered primitives, so `a = {}` and `b = []` is clear, whereas `Object.create(null)` is reaching into a part of the API that is unnecessary; in this case, also the use of `null` when JS already "passes in" `undefined` is generally considered making it less JS and more "writing C++/Java with JS syntax" (where you need explicit arguments even if what you need passed in is an undefined/null-reference)